metadata.json

```diff
@@ -85,8 +85,6 @@
     {
       "operatingsystem": "Ubuntu",
       "operatingsystemrelease": [
-        "12.04",
```

Comment:
These old platforms are only here for you, so feel free to remove anything that is end of life that you are no longer supporting.

Reply:
Thanks, good to know :)

Reply:
@anders-larsson please let me know which OS versions do you would like to get support for.

Reply:
*From our perspective on what can be removed*
* Solaris (not used at all. Maybe someone else does?)
* SLE[DS]10 (all other releases supported[1], SLED11 is not supported but SLES11 is with LTSS)
* Debian (probably same as Ubuntu though, keep it)
* Ubuntu 12.04 (looks like all other versions are still supported[2])

[1]: https://www.suse.com/c/explore-options-for-suse-linux-enterprise-11-service-pack-4-end-of-life-mar-31-2019/
[2]: https://ubuntu.com/about/release-cycle

Reply:
Thanks Anders ! Will re-add Ubuntu 14.04 later on.

Reply:
If Ubuntu 14.04 needs any specific tweaks we need to add them later. There isn't any specific handling in the code. Guessing that the default values are valid for it.

Reply:
> If Ubuntu 14.04 needs any specific tweaks we need to add them later. There isn't any specific handling in the code. Guessing that the default values are valid for it.

This could be useful https://github.com/kodguru/puppet-module-ssh/commit/1159fed26260babc7c9e663b0d092dfd9b9be16b

Reply:
Great and perfect timing. Just realised that the old unit tests doesn't verify Ubuntu 12 nor 14 and the code actually fails to run on them.

Are these values from a stock Ubuntu 14 system ? Or is it tweaked in any way ?

Reply:
I'm not 100% sure. I just re-added the old configuration that was previously present in this module.